test/functional/api/vim_spec.lua

```diff
@@ -1267,5 +1268,68 @@ describe('API', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
+  describe('statusline', function()
+    before_each(function()
+      screen = Screen.new(40, 4)
```

Comment:
What `screen` is used here? Should it be local to the `describe()` block?

Reply:
You are right, made it local to the describe scope.